app/src/main/res/layout/activity_detail.xml

```diff
@@ -20,20 +20,31 @@
             app:contentScrim="?attr/colorPrimary"
             app:layout_scrollFlags="scroll|exitUntilCollapsed">
 
-            <org.imozerov.streetartview.ui.detail.SquareImageView
-                android:id="@+id/art_object_detail_image"
```

Comment:
As I understood we do not have a collapsible toolbar now. Frankly said, that is a bit sad :)